### PR TITLE
revise orgs index view

### PR DESCRIPTION
**Pull request by Robertgervais**

### CODE REVIEW

**Comment by Robertgervais** [comment]
orgs_index

File: app/views/admin/organizations/index.html.haml [hunk 1]
```diff
@@ -1,33 +1,38 @@
...
 
-%section.full-screen-table
+.full-screen-table-wrapper
   %table.table.table-striped.table-bordered
-    %thead
+    %thead.thead-light
       - ['name', 'approved', 'created_at'].each do |col|
         - active_col = @sort == col
         %th
-          %a.column_sort_update{ href: '#', class: ('active' if active_col), style: 'display: block;', data: { sort: col, direction: (active_col && @sort_direction == 'desc' ? 'asc' : 'desc') } }
```
Comment (sethherr) [inline app/views/admin/organizations/index.html.haml]: This is its own legacy sortable functionality (check in `app/views/admin/organizations/index.html.haml`)

We have a new sortable functionality! Using a concern and some helpers. It's viewable in the paid_features controller index. This should be switched over to that!

**Comment by Robertgervais** [comment]
Image 4-15-19 at 4 22 PM

File: app/views/admin/organizations/index.html.haml [hunk 2]
```diff
@@ -1,76 +1,76 @@
...
+  .col-md-3
+    = link_to 'New Organization', new_admin_organization_url, class: "btn btn-success float-right"
 
 %h3
   #{number_with_delimiter(@organizations_count)} organizations
   = (params[:query].present? || params[:is_paid].present?) ? 'matching' : 'in the system'
 
-%div{ style: "width: 100%; text-align: right;" }
   = form_tag admin_organizations_path, method: :get do
-    = hidden_field_tag :sort, @sort
```
Comment (sethherr) [inline app/views/admin/organizations/index.html.haml]: When you search, your sorting should continue to be present.

File: app/views/admin/organizations/index.html.haml [hunk 3]
```diff
@@ -1,76 +1,76 @@
...
-            - if col == "created_at"
-              <small class="convertTimezone"></small>
-            - if active_col
-              .pull-right
-                %small
-                  = (@sort_direction == 'asc' ? '&#x2193;' : '&#x2191;').html_safe
+= paginate @organizations, views_prefix: "admin"
 
+.full-screen-table-wrapper
+  %table.table.table-striped.table-bordered
```
Comment (sethherr) [inline app/views/admin/organizations/index.html.haml]: `table-sm`

File: app/views/admin/organizations/index.html.haml [hunk 4]
```diff
@@ -1,76 +1,76 @@
...
               = organization.locations.count
               - if organization.show_on_map
                 %span.less-strong-right
                   Mapped
 
+  = paginate @organizations, views_prefix: "admin"
+
 
 :coffeescript
   $('.column_sort_update').click (e) ->
```
Comment (sethherr) [inline app/views/admin/organizations/index.html.haml]: This coffeescript was for the old sorting functionality, it shouldn't be in there anymore.

File: app/controllers/admin/organizations_controller.rb [hunk 5]
```diff
@@ -1,16 +1,17 @@
 class Admin::OrganizationsController < Admin::BaseController
+  include SortableTable
   before_filter :find_organization, only: [:show, :edit, :update, :destroy]
-  before_filter :set_sort_and_direction, only: [:index]
 
   def index
     page = params[:page] || 1
     per_page = params[:per_page] || 25
-    orgs = Organization.all
+    orgs = Organization.order(sort_column + " " + sort_direction).page(page).per(per_page)
```
Comment (sethherr) [inline app/controllers/admin/organizations_controller.rb]: Apply pagination once, and as the final thing (where you're applying it lower is the correct place)

File: app/views/admin/organizations/index.html.haml (hunk 2 above)
Comment (sethherr) [reply app/views/admin/organizations/index.html.haml]: You should test this functionality out and make sure it works.

**Pull request merged**